wgpu-core/Cargo.toml

```diff
@@ -44,9 +44,8 @@ smallvec = "1"
 thiserror = "1"
 
 [dependencies.naga]
-#git = "https://github.com/gfx-rs/naga"
-#rev = "27d38aae"
-version = "0.9"
```

Comment:
Nit: we can actually leave the version number here, this will let us I use a git revision locally, but still let us publish to crates.io